### ISSUE TITLE
problem with tf.placeholder

### ISSUE THREAD

**Issue by Lax94** [charlesq34/pointnet2#130]
when I run train.py it shows me this problem, how can I fix it?
![Cattura](https://user-images.githubusercontent.com/50701830/58623521-73d93780-82ce-11e9-88fa-3bcb0d1f55bb.PNG)

**Comment by sheshap** [comment]
Seems like you are using the pointnet model instead of pointnet++

Please check if you want to raise this issue in pointnet github link.

**Comment by oscar-lorente** [comment]
Did you solve it? I'm facing the same error...

**Comment by jroch01** [comment]
I think this is because you are using TensorFlow 2.X. To solve this you can add the following lines when importing TensorFlow:

import tensorflow.compat.v1 as tf
tf.disable_v2_behavior()

That should fix the tf.placeholder issue but you will run into similar problems later on, particularly with tf.contrib which this method won't work for. You may have to update the batch normalization method to get past that and change the initializer in tf_util.py to: 

initializer = tf.truncated_normal_initializer()
